src/software/ai/navigator/path_planner/theta_star_path_planner.h

```diff
@@ -344,8 +343,11 @@ class ThetaStarPathPlanner : public PathPlanner
     double max_navigable_x_coord;
     double max_navigable_y_coord;
 
-    // open_list represents Coordinates that we'd like to visit
-    std::set<Coordinate> open_list;
+    // open_list represents Coordinates that we'd like to visit. Elements are pairs of
+    // path_cost_and_end_dist_heuristic and Coordinate, so the set is implicitly ordered
```

Comment:
should `path_cost_and_end_dist_heuristic` be `sum_of_path_cost_and_end_dist_heuristic`? or `path_cost_plus_end_dist_heuristic`?

Reply:
That's a good point. This is thetastar's estimate of the total path cost if that cell is used, so I will change it to that.